### ISSUE TITLE
Maven problems

### ISSUE THREAD

**Issue by tfeiner** [spring-projects/spring-webflow-samples#39]
After cloning the repository, I executed "mvn package -U". The following error occurred:

```
[ERROR] Failed to execute goal on project booking-mvc: Could not resolve dependencies for project 
org.springframework.webflow.samples:booking-mvc:war:1.0.0.BUILD-SNAPSHOT: The following artifacts 
could not be resolved: org.springframework.webflow:spring-webflow:jar:2.4.3.RELEASE, 
org.springframework.webflow:spring-js:jar:2.4.3.RELEASE: Could not find artifact 
org.springframework.webflow:spring-webflow:jar:2.4.3.RELEASE in spring 
(http://repo.spring.io/snapshot) -> [Help 1]
```

**Comment by tfeiner** [comment]
should the repo be http://repo.spring.io/release instead of http://repo.spring.io/snapshot? (in projekt booking-mvc)?

**Comment by rstoyanchev** [comment]
Should be 2.4.4 (fixed now), 2.4.3 was a failed release. Thanks.
